py/server/deephaven/stream/__init__.py

```diff
@@ -11,6 +11,24 @@
 from deephaven.table import Table
 
 _JBlinkTableTools = jpy.get_type("io.deephaven.engine.table.impl.BlinkTableTools")
+_JAddOnlyToBlinkTableAdapter = jpy.get_type("io.deephaven.engine.table.impl.AddOnlyToBlinkTableAdapter")
+
+def add_only_to_blink(table: Table) -> Table:
+    """ Creates a blink table from the given add-only table.
```

Comment:
This needs a big note as to when you would use this. Really, it's only suitable when your add-only table is not fully in-memory, or when you have some other reason to want to opt into blink-specific aggregation semantics, typically because you're an expert in the query engine and trying to optimize state management.